Addding kraft installation for fedora and other dnf-distributions. (#60)
unikraft/docs: content/en/docs/getting-started/_index.md

  unzip \
  uuid-runtime \
  python3 \
  python3-setuptools \
  python3-pip
```

> ## Installing `kraft` on Fedora and other dnf-based-systems

Reviewer: I would not add a title, since there is no separate title for the above command.
You can just rephrase the next phrase to make it an introduction.

> Please run following commands to install `kraft` on Fedora and other dnf-based systems.

Reviewer: ```suggestion
Please run the following commands to install `kraft` on dnf-based systems.
```

```bash

> sudo dnf install @development-tools ncurses-devel libyaml flex git wget socat bison unzip libuuid python3 python3-setuptools

Reviewer: I would do the same layout as above, meaning one required component per line.
